pkg/p2p/libp2p/internal/handshake/handshake.go

```diff
@@ -24,16 +27,22 @@ const (
 	ProtocolName    = "handshake"
 	ProtocolVersion = "1.0.0"
 	StreamName      = "handshake"
-	messageTimeout  = 5 * time.Second // maximum allowed time for a message to be read or written.
+	messageTimeout  = 10 * time.Second // maximum allowed time for a message to be read or written.
```

Comment:
is this increase really needed?

Reply:
I was testing locally during debugging, so maybe not, I will switch it back and increase if needed.